### ISSUE TITLE
Could not connect to your TREZOR. Verify cable ....

### ISSUE THREAD

**Issue by waratoyo** [dashpay/electrum-dash-old#82]
I would like to share my experience with the electrum-dash 2.6.4 on Ubuntu 16.04.
If you experience the same issue with TREZOR connected follow the instructions. 
It works for me until developers find way how to resolve the issue. 

in module /plugins/trezor/plugin.py replace part of the code with:

  def create_client(self, device, handler):
        from trezorlib.transport_hid import HidTransport

```
    devices = HidTransport.enumerate()
    if len(devices) == 0:
           print('No TREZOR found')
           return None
    transport = HidTransport(devices[0])

    self.print_error("connected to device at", device.path)

    client = self.client_class(transport, handler, self)

    # Try a ping for device sanity
```

**Comment by schinzelh** [comment]
Thanks for reporting, I'll havea look at this. The USB/HID code is very fragile it seems...

**Comment by Kefkius** [comment]
Updating from python-trezor 0.6.13 to 0.7.0 causes this problem for me on Ubuntu 16.04. Updating python-trezor also updated hidapi from 0.7.99-5 to 0.7.99.post18. Working on a fix

**Comment by Kefkius** [comment]
Fixed in #83. Calling HidTransport() resulted in an instance method call, which resulted in the Plugin instance being the first argument instead of the pair.

**Comment by nmarley** [comment]
Hi, the current Electrum Dash fork is maintained here: https://github.com/akhavr/electrum-dash

Please use that repo and report there if you have any issues.
